scripts/lynx/authorize_enrico.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+
+
+from ape import project
+from eth_utils import to_checksum_address
+
+from deployment.constants import ARTIFACTS_DIR
+from deployment.params import Transactor
+from deployment.registry import contracts_from_registry
+from deployment.utils import check_plugins
+
+REGISTRY_FILEPATH = ARTIFACTS_DIR / "lynx.json"
+
+
+def main():
+    check_plugins()
+    transactor = Transactor()
+    deployments = contracts_from_registry(filepath=REGISTRY_FILEPATH)
+    global_allow_list = deployments[project.GlobalAllowList.contract_type.name]
+    ritual_id = int(input("Enter ritual ID: "))
+    addresses = [to_checksum_address(input("Enter address to authorize: "))]
+    transactor.transact(global_allow_list.authorize, ritual_id, addresses)
```

Comment:
Maybe this could be done as part of #154 as well, but should we check global_allow_list.isAddressAuthorized(...) first to potentially save some gas in case enrico is already authorized. 

Similar to what we do here - https://github.com/nucypher/nucypher/blob/development/scripts/hooks/nucypher_dkg.py#L336.

Just something to think about.